modules/gapi/src/backends/ocl/goclcore.cpp

```diff
@@ -523,6 +542,82 @@ GAPI_OCL_KERNEL(GOCLTranspose, cv::gapi::core::GTranspose)
     }
 };
 
+GAPI_OCL_KERNEL(GOCLBGR, cv::gapi::streaming::GBGR)
+{
+    static void run(const cv::MediaFrame& in, cv::UMat& out)
+    {
+        cv::util::suppress_unused_warning(in);
+        cv::util::suppress_unused_warning(out);
+#ifdef HAVE_DIRECTX
+#ifdef HAVE_D3D11
+#ifdef HAVE_ONEVPL
+        auto d = in.desc();
+        if (d.fmt != cv::MediaFormat::NV12)
+        {
+            GAPI_LOG_DEBUG(nullptr, "Unsupported format provided: " << d.fmt <<
+                           ". Expected cv::MediaFormat::NV12.");
```

Comment:
It must be log error severity, not debug
Here and below

Reply:
There is no `GAPI_LOG_ERROR` only `FATAL`. Should it be `FATAL`?

Reply:
at least it shouldn't be debug

Reply:
Done